### DISCUSSION TITLE
Removing attachments permissions from User Edit page

### DISCUSSION THREAD

**Discussion by jfuentesa** [orchidsoftware/platform#2605] [Q&A] [answered]
Hi:

I have a question regarding the permissions in the Orchid backend. Is there any way to remove the "Attachments" permission from appearing on the User Edit Page in the backend?

I would like to eliminate the option because I don't use it in my application.

Thank you.

**Comment by tabuna** [comment, marked as answer]
You can use the method:

```php
Dashboard::removePermission('need key')
```

It is best to do this after registration, for example with a service provider

**Reply by smartyaunt** [reply]
where get the key, key mean platform.systems.attachment?

**Reply by smartyaunt** [reply]
I checked the code and there seems to be a bug in the remove method
